app/helpers/index_helper.rb

```diff
@@ -0,0 +1,17 @@
+module IndexHelper
+  def title_and_url(flow, presenter)
+    tag.p(presenter.start_node.title) + link_to("/#{flow.name}", smart_answer_path(flow.name))
+  end
```

Comment:
To be consistent with the other helpers, I think flow name should be passed into this method rather than the `flow` object.